### ISSUE TITLE
New Gmail login screen directs to empty

### ISSUE THREAD

**Issue by jordan-rothenheimer** [vain/lariza#45]
When I go to `mail.google.com`, I click "sign in" and enter my e-mail and password when prompted. I am then redirected to `https://accounts.google.com/signin/v2/challenge/password/empty`, which is just an empty screen. Reloading does not help, and when I run `lariza` from a terminal it does not produce any weird error messages on standard output.

I can log in using `google-chrome-stable`, but `links` gets stuck in a loop and prompts me for my e-mail after I enter my password, and `surf` takes me to `https://accounts.google.com/AccountChooser?service=mail&continue=https://mail.google.com/mail/`, which is another empty screen.

I figured out how to download e-mails onto my computer for browsing, which I had been meaning to do anyway so this malfunction has actually been pretty great for me ... anyway I do not consider this an urgent issue. All my other webmails work. I am on Arch Linux.

**Comment by vain** [comment]
This appears to be a webkit "bug":

https://bugs.webkit.org/show_bug.cgi?id=171770

Note that, just today, a discussion started about this topic on the webkit-dev mailing list:

https://lists.webkit.org/pipermail/webkit-dev/2017-May/028991.html

tl;dr: There will probably be a new webkit release shortly which contains a "fix" for this.

The only thing I could do about this in lariza would be enabling users to enable/disable "site specific quirks":

https://webkitgtk.org/reference/webkit2gtk/stable/WebKitSettings.html#WebKitSettings--enable-site-specific-quirks

But I won't be going that path. It might "fix" some web sites, but it will break others.

This is madness.
